src/components/tx/create.js

```diff
@@ -141,30 +142,37 @@ const CreateTx = connect(
     },
     (dispatch, ownProps) => {
         return {
-            onSubmit: data => {               
+            onSubmit: data => {
+                const afterTx = (txhash) => {
+                    let txdetails = {
+                        hash: txhash,
+                        account: ownProps.account
+                    };
+                    dispatch(trackTx(txhash));
+                    dispatch(gotoScreen('transaction', txdetails));
+                };
+                const resolver = (resolve, f) => {
```

Comment:
what's the difference between this and `.then((response) => resolve(response));`? just curious